entity-framework/core/modeling/inheritance.md

```diff
@@ -49,8 +46,41 @@ Finally, the discriminator can also be mapped to a regular .NET property in your
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/NonShadowDiscriminator.cs?name=NonShadowDiscriminator&highlight=4)]
 
-## Shared columns
+### Shared columns
```

Comment:
Unrelated to docs question, what happens if user decides to change type of one of them? (let's say the change which is non-destructive)

Reply:
Validation will fail

Reply:
Found my mistake. I thought we merged automatically. I read the docs again and we allow user to configure it so no magical change.